src/parser.js

```diff
@@ -1480,43 +1473,53 @@ export default class Parser {
     return hrefVariables.length ? hrefVariables : undefined;
   }
 
-  // Create a Refract asset element containing JSON Schema and push into payload
-  pushSchemaAsset(schema, payload, path) {
-    let handledSchema = false;
+  pushBodyAssets(schema, payload, contentType, pushBody) {
+    let jsonSchema;
 
     try {
-      const jsonSchema = convertSchema(schema);
-      const Asset = this.minim.getElementClass('asset');
-      const schemaAsset = new Asset(JSON.stringify(jsonSchema));
-
-      schemaAsset.classes.push('messageBodySchema');
-      schemaAsset.contentType = 'application/schema+json';
-
-      if (this.generateSourceMap) {
-        this.createSourceMap(schemaAsset, path);
-      }
-
-      this.handleExternalDocs(schemaAsset, schema.externalDocs);
-
-      payload.content.push(schemaAsset);
-      handledSchema = true;
```

Comment:
What happened to the logic this variable is being used for?

Reply:
There is a return on line 1486.